app/javascript/utilities/__tests__/localDateTime.test.js

```diff
@@ -1,26 +1,41 @@
-import {timestampToLocalDateTime, addLocalizedDateTimeToElementsTitles } from '@utilities/localDateTime';
+import {
+  timestampToLocalDateTime,
+  addLocalizedDateTimeToElementsTitles,
+} from '@utilities/localDateTime';
 
 describe('LocalDateTime Utilities', () => {
   it('should return empty string when no timestamp', () => {
-    const localTime = timestampToLocalDateTime(null, null, null)
+    const localTime = timestampToLocalDateTime(null, null, null);
     expect(localTime).toEqual('');
   });
 
   it('should return readable date string', () => {
-    const localTime = timestampToLocalDateTime('2019-05-03T16:02:50.908Z', 'default', {})
+    const localTime = timestampToLocalDateTime(
+      '2019-05-03T16:02:50.908Z',
+      'en',
```

Comment:
This PR contains some indent changes due to Prettier command, but I only changed this line and [24](https://github.com/forem/forem/pull/20230/files#diff-5cc57955ad882c82a15d33a10070ee9a78489371358efda3750a55c76b1fdd3dR24).